src/components/ScopedTheme.tsx

```diff
@@ -21,12 +21,8 @@ export const ScopedTheme: React.FunctionComponent<React.PropsWithChildren<ThemeP
     const mappedChildren = [
         <Apply key={name} name={name} />,
         children,
-        <Apply key='dispose' name={previousScopedTheme as keyof UnistylesThemes | undefined} />
+        <Apply key="dispose" name={previousScopedTheme as keyof UnistylesThemes | undefined} />,
     ]
 
-    return (
-        <React.Fragment>
-            {mappedChildren}
-        </React.Fragment>
-    )
+    return <>{mappedChildren}</>
```

Comment:
Only line where I fixed a biome lint warning

Reply:
Can we force <React.Fragment>, I'm not a big fan of `<></>`

Reply:
Alright, in this case the warning was because the fragment was unecessarry so I removed it